### PR TITLE
HARVESTER: fix harvester plugin rc version

### PR DESCRIPTION
**Pull request by WuJun2016**
### Summary
Fixes #
- https://github.com/harvester/harvester/issues/3145

### Technical notes summary

Now we will build the following 3 types of harvester-plugin address
1.  master-head:   https://releases.rancher.com/harvester-ui/plugin/harvester-latest/harvester-latest.umd.min.js
2. release:   https://releases.rancher.com/harvester-ui/plugin/harvester-${tag-version}/harvester-${tag-version}.umd.min.js
e.g: https://releases.rancher.com/harvester-ui/plugin/harvester-v1.0.1-rc10/harvester-v1.0.1-rc10.umd.min.js
3.  branch-head:    https://releases.rancher.com/harvester-ui/plugin/harvester-${branch-name}/harvester-${branch-name}.umd.min.js
e.g:  https://releases.rancher.com/harvester-ui/plugin/harvester-release-harvester-v1.1/harvester-release-harvester-v1.1.umd.min.js

### CODE REVIEW

**Comment by WuJun2016** [comment]
@guangbochen   @futuretea   PTAL, thanks.

File: shell/scripts/drone-build-pkg.sh [hunk 1]
```diff
@@ -20,13 +20,13 @@ echo "COMMIT_BRANCH: ${COMMIT_BRANCH}"
...
 if [ -n "$GIT_TAG" ]; then
-  TAG_VERSION=$(cd pkg/$1; node -p -e "require('./package.json').version")
-  COMMIT=$COMMIT COMMIT_BRANCH=$COMMIT_BRANCH VERSION=$TAG_VERSION ./shell/scripts/build-pkg.sh ${1} "true"
+  COMMIT=$COMMIT COMMIT_BRANCH=$COMMIT_BRANCH VERSION=$DRONE_TAG ./shell/scripts/build-pkg.sh ${1} "true"
+else
+  COMMIT=$COMMIT COMMIT_BRANCH=$COMMIT_BRANCH VERSION=$VERSION ./shell/scripts/build-pkg.sh ${1} "true"
 fi
 
 # package, override version, commit for file in pkg root
 # Note - in the future env vars should be moved to args and build-pkg should use getopts
```
Comment (futuretea) [inline shell/scripts/drone-build-pkg.sh:29]: The position of the comment does not match the position of the code

Comment (WuJun2016) [reply shell/scripts/drone-build-pkg.sh:29]: I'm going to delete these two lines of comments

Comment (WuJun2016) [reply shell/scripts/drone-build-pkg.sh]: Done

**Review by futuretea — APPROVED** [review]
LGTM

**Review by guangbochen — APPROVED** [review]
LGTM, thanks.

**Comment by guangbochen** [comment]
Do we need to backport this PR to the master branch?

**Comment by WuJun2016** [comment]
> 

Yes, https://github.com/harvester/dashboard/pull/509

**Pull request merged**